src/ReinforcementLearningZoo/src/algorithms/offline_rl/CQL_SAC.jl

```diff
@@ -0,0 +1,110 @@
+import LogExpFunctions
+export CQLSACPolicy
+
+"""
+    CQLSACPolicy(
+        sac::SACPolicy,
+        action_sample_size::Int = 10,
+        α_cql::Float32 = 0f0,
+        α_lr::Float32 = 1f-3,
+        τ_cql::Float32 = 5f0,
+        α_cql_autotune::Bool = true,
+        cons_weight::Float32 = 1f0, #multiplies the Q difference before substracting tau, used to scale with the rewards.
+        finetune_experiment::E = nothing #Provide an second experiment to run at PostExperimentStage to finetune the sac policy, typically with an agent that uses the sac policy. Leave nothing if no finetuning is desired.
+    )
+
+    Implements the Conservative Q-Learning algorithm [1] in its continuous variant on top of the SAC algorithm [2]. `CQLSACPolicy` wraps a classic `SACPolicy` whose networks will be trained normally, except for the additional conservative loss.
+    CQLSACPolicy contains the additional hyperparameters that are specific to this method. α_cql is the lagrange penalty for the conservative_loss, it will be automatically tuned if ` α_cql_autotune = true`. `cons_weight` is a scaling parameter 
```

Comment:
```suggestion
    `CQLSACPolicy` contains the additional hyperparameters that are specific to this method. α_cql is the lagrange penalty for the conservative_loss, it will be automatically tuned if ` α_cql_autotune = true`. `cons_weight` is a scaling parameter 
```